editor.planx.uk/src/@planx/components/FindProperty/Public/index.tsx

```diff
@@ -33,6 +34,18 @@ export const FETCH_BLPU_CODES = gql`
 
 type Props = PublicProps<FindProperty>;
 
+const AddressLoadingWrap = styled(Box)(({ theme }) => ({
+  marginBottom: theme.spacing(-2.5),
+  minHeight: theme.spacing(3),
+  pointerEvents: "none",
+  [theme.breakpoints.up("md")]: {
+    position: "absolute",
+    bottom: theme.spacing(1),
+    margin: 0,
+    paddingRight: theme.spacing(14),
+  },
+}));
```

Comment:
nit: this works perfectly on full screen routes, but if I use FindProperty from the editor side panel the loading indicator is overlapping the disabled Continue button

Reply:
Good catch, I've updated to be aligned left and padded to leave space for the continue button, a more universal solution.

Reply:
Working perfectly, let's merge :rocket: